components/EventPartners.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Photo from './Photo';
+
+const EventPartners = ({ event }) => {
+  return (<section className="mb-6">
```

Comment:
No need for the return here - remove the 
```suggestion
const EventPartners = ({ event }) => {
  return (<section className="mb-6">
```
```suggestion
const EventPartners = ({ event }) => (
  <section className="mb-6">
```